docs/topics/proc_getting-the-secured-booster-api-endpoint.adoc

```diff
@@ -33,10 +33,10 @@ A sample output is shown in the following table:
 | <all>
 | passthrough
 
-| secured-{parameter-runtime}-http
-| secured-{parameter-runtime}-http-myproject.LOCAL_OPENSHIFT_HOSTNAME
+| {parameter-runtime}-rest-http-secured
```

Comment:
if `{parameter-runtime}` is `spring-boot` for the Spring Boot runtime, then yes. Hmm, on second thought, maybe not. This value is actually the artifact Id of the Maven project. While this value is correct for the Spring Boot runtime  booster *before* Launcher takes it and "clones" it with the user having the possibility to change the artifact id. So in fact, we don't know what that value may be until runtime…

Reply:
in this case, the parameter is "sb".

what if we changed the whole thing to <NAME_OF_ARTIFACT> or something similar?

Reply:
`sb` is not correct, then. But in any case, this needs to be changed. The problem with using <NAME_OF_ARTIFACT> is that we don't currently have no explanation of the launch process to reference and explain that the artifact id is what was chosen when the booster was created in Launch (or the default value generated by Launch, which is *not* the value that we set in the boosters)… This needs to be well documented and it is not trivial to do so.